2024-01-02 13:21:58,831 ERROR: Exception on /api/users/3/chats/1 [PUT] [in /home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py : 825]
Traceback (most recent call last):
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/app/api/users.py", line 32, in update_chat
    conversation.from_dict(id,data=data)
  File "/home/py_sama/rapid_care/app/models.py", line 188, in from_dict
    return_link = {search[i][0].all()[0].name:search[i][0].all()[0].url for i in range(len(search)) if search[i][1]['value']!=0}
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/app/models.py", line 188, in <dictcomp>
    return_link = {search[i][0].all()[0].name:search[i][0].all()[0].url for i in range(len(search)) if search[i][1]['value']!=0}
                                                                                                       ~~~~~~~~~~~~^^^^^^^^^
TypeError: 'int' object is not subscriptable
2024-01-02 13:25:12,420 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:25:12,420 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:25:12,420 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:25:12,420 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:27:40,152 ERROR: Exception on /api/users/3/chats/3 [PUT] [in /home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py : 825]
Traceback (most recent call last):
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/rapid_care_environment/lib/python3.11/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/app/api/users.py", line 32, in update_chat
    conversation.from_dict(id,data=data)
  File "/home/py_sama/rapid_care/app/models.py", line 189, in from_dict
    return_link = {search[i] for i in range(len(search))}
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/py_sama/rapid_care/app/models.py", line 189, in <setcomp>
    return_link = {search[i] for i in range(len(search))}
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: unhashable type: 'list'
2024-01-02 13:28:40,341 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:28:40,341 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:28:40,346 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:28:40,346 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:31:30,449 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:31:30,450 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:31:30,451 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:31:30,451 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:32:24,353 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:32:24,364 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:32:24,457 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:32:24,477 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:37:24,590 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:39:33,176 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:39:33,204 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:39:33,218 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:39:33,218 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:40:09,483 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:42:22,368 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:42:22,368 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:42:22,368 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:42:22,368 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:43:07,324 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:43:07,332 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:43:07,394 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:43:07,443 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:48:50,199 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:49:08,214 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:53:09,121 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:58:40,663 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:58:40,663 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:58:40,672 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 13:58:40,672 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:00:46,758 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:04:27,983 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:08:05,774 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:08:05,812 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:08:05,832 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:08:05,932 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:52:20,556 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:52:20,571 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:52:20,743 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:54:04,450 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:54:04,646 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:54:04,763 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 14:54:04,783 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 15:06:53,361 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 15:06:53,421 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 15:06:53,465 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 15:06:53,514 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 16:41:39,349 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 83]
2024-01-02 16:42:42,229 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 83]
2024-01-02 16:43:49,157 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 83]
2024-01-02 16:44:52,505 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 83]
2024-01-02 16:49:03,645 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 16:51:06,362 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
2024-01-02 20:04:05,431 INFO: hub_startup [in /home/py_sama/rapid_care/app/__init__.py : 80]
